serverless-heaven/serverless-webpack
# fix for dependencies with no version detected

## What did you implement:

Closes #414

## How did you implement it:

Changed yarn `--frozen-lockfile` to `--pure-lockfile`. This will still install the package when no version is detected.

## How can we verify it:

1. Add a dependency that is not a direct dependency to the externals.
2. Set configuration to yarn.
3. run `sls webpack`
4. See that the packages are installed and the process doesn't exit with an error.

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
